Articles/Blog/HelloHeatEquation.md

```diff
@@ -1,143 +1,99 @@
+ - <img src='https://github.com/betterscientificsoftware/images/raw/master/hello_world_sun_heat.png'/>
 
-## What is the Heat Equation?
-The Heat Equation, also known as the Diffusion Equation, was developed by Joseph Fourier in 1822. Primarily used in the fields of Mathematics and Physics, the equation shows how heat (or something similar) changes over time in a solid medium. While there are many adaptations for different dimensions and scenerios, new programmers are often first introduced to the 1-dimensional version of the Heat Equation as a tutorial problem.
-
-The following description is adapted from [a Trinity University lecture](http://ramanujan.math.trinity.edu/rdaileda/teach/s14/m3357/lectures/lecture_2_25_slides.pdf "One-Dimensional Heat Equation").
-
-Suppose you have a thin metal rod along the x-axis starting at position 0 and ending at position _L_. The goal is to model the temperature distribution throughout the rod at a given time, with the following (ideal) assumptions:
+#### Contributed by [Heather M. Switzer](https://github.com/heatherms27 "Heather Switzer GitHub Profile"), [Elsa Gonsiorowski](https://github.com/gonsie "Elsa Gonsiorowski GitHub Profile") and [Mark C Miller](https://github.com/markcmiller86 "Mark C Miller GitHub Profile")
 
-* An initial temperature distribution is given
-* Boundary conditions (established tempuratures at each end of the rod) are known
-* Heat only moves horizontally through the bar
-* There are no external heating/cooling sources
+["Hello World!"](https://www.thesoftwareguild.com/blog/the-history-of-hello-world/)
+is the canonical *first* program beginners write to learn new programming methodologies.
+In computational science and engineering ([CSE](https://en.wikipedia.org/wiki/Computational_engineering))
+circles, programming methodologies can involve a variety of languages, parallel execution
+and programming models, discretization techniques, interdependent libraries and more.
+An application solving the [heat equation](https://en.wikipedia.org/wiki/Heat_equation)
+represents the perfect "Hello World!" program for the CSE world.
 
-<div style="text-align:center"><img width="75%" src="https://github.com/betterscientificsoftware/images/blob/master/Blog_0719_HeatEqnBar.png" /></div>
-
-To model the temperature distribution of the rod, we can use the 1-dimensional Heat Equation given by the formula:
+## What is the Heat Equation?
+The Heat Equation, a special case of the Diffusion Equation, was developed by Joseph
+Fourier in 1822. It describes how the distribution of heat varies over space and time
+in a material. While there are many adaptations for different scenarios, beginner CSE
+programmers often start with the one-dimensional, homogeneous Heat Equation as an
+introductory problem.
 
```

Comment:
please add an introduction to the equation. Something like "The general heat equation is:"